examples/kubernetes/configurator-values.yaml

```diff
@@ -0,0 +1,541 @@
+      # @default -- `""`
+      # ca_file:
+
+      # -- Certificate and key files path for client cert authentication to the server.
+      # @default -- `""`
+      # cert_file:
+      # key_file:
+
+      # Disable validation of the server certificate.
+      # @default -- `false`
+      # insecure_skip_verify:
+
+    # -- Sets the `Authorization` Bearer token header on every scrape request
+    # @default -- `{}`
+    # authorization:
+      # Sets the credentials to the credentials read from the configured file.
+      # @default -- `""`
+      # credentials_file:
+
+    # -- Sets the `Authorization` header on every scrape request with the configured username and password.
+    # @default -- `{}`
+    # basic_auth:
+      # username:
+      # password_file:
+
+    # -- List of relabeling configurations. Used if needed to add any special filter or label manipulation before the scrape takes place.
+    # @default -- `[]`
+    # extra_relabel_config:
+
+    # -- List of metric relabel configurations. Used it to filter metrics and labels after scrape.
+    # @default -- `[]`
+    # extra_metric_relabel_config:
+
+
+  # -- It allows defining scrape jobs for targets with static URLs.
+  # @default -- See `values.yaml`.
+  static_targets:
+    # -- List of static target jobs. By default, it defines a job to get self-metrics. Please note, if you define `static_target.jobs` and would like to keep
+    # self-metrics you need to include a job like the one defined by default.
+    # @default -- See `values.yaml`.
+    jobs:
+    - job_name: self-metrics
+      skip_sharding: true  # sharding is skipped to obtain self-metrics from all Prometheus servers.
+      targets:
+        - "localhost:9090"
+      extra_metric_relabel_config:
+        - source_labels: [__name__]
+          regex: "\
+            prometheus_agent_active_series|\
+            prometheus_target_interval_length_seconds|\
+            prometheus_target_scrape_pool_targets|\
+            prometheus_remote_storage_samples_pending|\
+            prometheus_remote_storage_samples_in_total|\
+            prometheus_remote_storage_samples_retried_total|\
+            prometheus_agent_corruptions_total|\
+            prometheus_remote_storage_shards|\
+            prometheus_sd_kubernetes_events_total|\
+            prometheus_agent_checkpoint_creations_failed_total|\
+            prometheus_agent_checkpoint_deletions_failed_total|\
+            prometheus_remote_storage_samples_dropped_total|\
+            prometheus_remote_storage_samples_failed_total|\
+            prometheus_sd_kubernetes_http_request_total|\
+            prometheus_agent_truncate_duration_seconds_sum|\
+            prometheus_build_info|\
+            process_resident_memory_bytes|\
+            process_virtual_memory_bytes|\
+            process_cpu_seconds_total"
+          action: keep
+
+# New Relic: Scrape Configs
+
+# Job to pull metrics from KSM
+    - job_name: kube-state-metrics
```

Comment:
Why do we need a static job if KSM has a service? There would be the risk of scraping it twice due to the jobs under the the kubernetes section

Reply:
The data didn't show up without both parts there, are you suggesting to omit the service part?